Error while parsing arguments: Unrecognized option: -vt

When running `make runPrePro` or `make run` in the psi demo, the following error appears in the logs:

```
Exception in thread "main" java.lang.IllegalArgumentException: Error while parsing arguments: Unrecognized option: -vt
        at dk.alexandra.fresco.demo.cli.CmdLineUtil.parse(CmdLineUtil.java:273)
        at dk.alexandra.fresco.demo.PrivateSetDemo.main(PrivateSetDemo.java:109)
Caused by: org.apache.commons.cli.UnrecognizedOptionException: Unrecognized option: -vt
```

Removing the -vt flag from the Makefile seems to work.

--- a/demos/psi/Makefile
+++ b/demos/psi/Makefile
@@ -9,13 +9,13 @@ build:
 	cp target/fresco-demo-psi.jar server2;
 
 runPrePro:
-	cd server1 && java -jar fresco-demo-psi.jar -e SEQUENTIAL_BATCHED -i 1 -l INFO -p 1:localhost:8081 -p 2:localhost:8082 -s tinytablesprepro -vt 1 -in 2,3,4,5,8,9,14 -key abc123abc123abc123abc123abc123ab > log.txt 2>&1 &
-	cd server2 && java -jar fresco-demo-psi.jar -e SEQUENTIAL_BATCHED -i 2 -l INFO -p 1:localhost:8081 -p 2:localhost:8082 -s tinytablesprepro -vt 1 -in 2,3,4,6,7,12,14 -key abc123abc123abc123abc123abc123ab > log.txt 2>&1 &
+	cd server1 && java -jar fresco-demo-psi.jar -e SEQUENTIAL_BATCHED -i 1 -l INFO -p 1:localhost:8081 -p 2:localhost:8082 -s tinytablesprepro -in 2,3,4,5,8,9,14 -key abc123abc123abc123abc123abc123ab > log.txt 2>&1 &
+	cd server2 && java -jar fresco-demo-psi.jar -e SEQUENTIAL_BATCHED -i 2 -l INFO -p 1:localhost:8081 -p 2:localhost:8082 -s tinytablesprepro -in 2,3,4,6,7,12,14 -key abc123abc123abc123abc123abc123ab > log.txt 2>&1 &
 
 run:
-	cd server1 && java -jar fresco-demo-psi.jar -e SEQUENTIAL_BATCHED -i 1 -l INFO -p 1:localhost:8081 -p 2:localhost:8082 -s tinytables -vt 1 -in 2,3,4,5,8,9,14 -key abc123abc123abc123abc123abc123ab > log.txt 2>&1 &
-	cd server2 && java -jar fresco-demo-psi.jar -e SEQUENTIAL_BATCHED -i 2 -l INFO -p 1:localhost:8081 -p 2:localhost:8082 -s tinytables -vt 1 -in 2,3,4,6,7,12,14 -key abc123abc123abc123abc123abc123ab > log.txt 2>&1 &
+	cd server1 && java -jar fresco-demo-psi.jar -e SEQUENTIAL_BATCHED -i 1 -l INFO -p 1:localhost:8081 -p 2:localhost:8082 -s tinytables -in 2,3,4,5,8,9,14 -key abc123abc123abc123abc123abc123ab > log.txt 2>&1 &
+	cd server2 && java -jar fresco-demo-psi.jar -e SEQUENTIAL_BATCHED -i 2 -l INFO -p 1:localhost:8081 -p 2:localhost:8082 -s tinytables -in 2,3,4,6,7,12,14 -key abc123abc123abc123abc123abc123ab > log.txt 2>&1 &
 
 runDummy:
-	cd server1 && java -jar fresco-demo-psi.jar -e SEQUENTIAL_BATCHED -i 1 -l INFO -p 1:localhost:8081 -p 2:localhost:8082 -s dummyBool -vt 1 -in 2,3,4,5,8,9,14 -key abc123abc123abc123abc123abc123ab > log.txt 2>&1 &
-	cd server2 && java -jar fresco-demo-psi.jar -e SEQUENTIAL_BATCHED -i 2 -l INFO -p 1:localhost:8081 -p 2:localhost:8082 -s dummyBool -vt 1 -in 2,3,4,6,7,12,14 -key abc123abc123abc123abc123abc123ab > log.txt 2>&1 &
+	cd server1 && java -jar fresco-demo-psi.jar -e SEQUENTIAL_BATCHED -i 1 -l INFO -p 1:localhost:8081 -p 2:localhost:8082 -s dummyBool -in 2,3,4,5,8,9,14 -key abc123abc123abc123abc123abc123ab > log.txt 2>&1 &
+	cd server2 && java -jar fresco-demo-psi.jar -e SEQUENTIAL_BATCHED -i 2 -l INFO -p 1:localhost:8081 -p 2:localhost:8082 -s dummyBool -in 2,3,4,6,7,12,14 -key abc123abc123abc123abc123abc123ab > log.txt 2>&1 &

